apps/Gruntfile.js

```diff
@@ -819,6 +820,38 @@ describe('entry tests', () => {
                 return chunkNames.includes(chunk.name);
               },
               priority: 20
+            },
+            vendors: {
+              name: 'vendors',
+              priority: 30,
```

Comment:
What does this do?

Reply:
we discussed offline briefly. more details are here: https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkscachegroupscachegrouppriority